crates/block-producer/src/runner.rs

```diff
@@ -423,11 +428,31 @@ pub fn run(config: Config, skip_config_check: bool) -> Result<()> {
                 Arc::clone(&poa),
                 base.store.clone(),
             );
+            let pg_pool = {
+                let config = config.web3_indexer.as_ref();
+                let init_pool = config.map(|web3_indexer_config| {
+                    smol::block_on(async {
+                        let mut opts: PgConnectOptions =
+                            web3_indexer_config.database_url.parse()?;
+                        opts.log_statements(log::LevelFilter::Debug)
+                            .log_slow_statements(log::LevelFilter::Warn, Duration::from_secs(5));
+                        PgPoolOptions::new()
+                            .max_connections(5)
```

Comment:
I am not sure if max_connections(5) is enough.

Reply:
I think it is sufficient if there are few time-consuming connections.
So, let's collect the `slow_statements` logs and make a decision later.